File: app/src/main/java/com/onedelay/mymovie/MyMovieApplication.java
package com.onedelay.mymovie;

import android.app.Application;

import com.android.volley.toolbox.Volley;
import com.onedelay.mymovie.api.VolleyHelper;

public class MyMovieApplication extends Application {
    @Override
    public void onCreate() {
        super.onCreate();
        VolleyHelper.requestQueue = Volley.newRequestQueue(getBaseContext());
    }
}

File: app/src/main/java/com/onedelay/mymovie/MyMovieApplication.kt
package com.onedelay.mymovie

import android.app.Application

import com.android.volley.toolbox.Volley
import com.onedelay.mymovie.data.api.VolleyHelper

class MyMovieApplication : Application() {
    override fun onCreate() {
        super.onCreate()
        VolleyHelper.requestQueue = Volley.newRequestQueue(baseContext)
    }
}
